pkg/subctl/cmd/deploybroker.go

```diff
@@ -42,6 +46,13 @@ func init() {
 	deployBroker.PersistentFlags().BoolVar(&disableDataplane, "no-dataplane", true,
 		"Don't install the Submariner dataplane on the broker (default)")
 	lighthouse.AddFlags(deployBroker, "service-discovery")
+
+	deployBroker.PersistentFlags().BoolVar(&globalnetEnable, "globalnet-enable", false,
+		"Enable support for Overlapping CIDRs in connecting clusters")
+	deployBroker.PersistentFlags().StringVar(&globalnetCidrRange, "globalnet-cidr-range", "169.254.0.0/16",
+		"Global CIDR supernet range for allocating GlobalCIDRs to each cluster")
+	deployBroker.PersistentFlags().Uint8Var(&globalnetClusterSize, "globalnet-cluster-size", 8,
```

Comment:
Those defaults are problematic, it means that each cluster can only have 256 pods+services.

Reply:
IPv4 Link-local address is 169.254.0.0/16, so we cannot expand it further.
I think we can do the following, the default value of globalnet-cidr-range can be 169.254.0.0/16 whereas globalnet-cluster-size can be modified to 19 bits.

With this configuration, we can have 8 connected clusters via Submariner and each cluster can have 8K Pods+Services (which I feel is a decent value for defaults). Thoughts?

Reply:
If user wants to override the defaults, instead of asking the input in networkMask/bits for the cluster, let the user specify the clustersize (i.e., typical number of Pods/Services that he intends to deploy in his cluster). 

For example:
If a user specifies that his clusterSize would be 200, subctl can allocate a globalCIDR of "/8" for that cluster.
If a user specifies his clusterSize is 7500, subctl would allocate the nearest upper bound - i.e., "/19"
and so on...

Reply:
The way I see these flags in use...
globalnet-enable and globalnet-cidr-range are applicable during Broker installation.
globalnet-cluster-size is applicable when joining data clusters.

Reply:
A clarification: globalnet-cluster-size is also applicable during broker installation, but it can be overridden during subctl join operation.